libs/community/langchain_community/embeddings/voyageai.py

```diff
@@ -69,15 +69,15 @@ class VoyageEmbeddings(BaseModel, Embeddings):
 
             from langchain_community.embeddings import VoyageEmbeddings
 
-            voyage = VoyageEmbeddings(voyage_api_key="your-api-key")
+            voyage = VoyageEmbeddings(voyage_api_key="your-api-key", model="voyage-2")
             text = "This is a test query."
             query_result = voyage.embed_query(text)
     """
 
-    model: str = "voyage-01"
+    model: str
```

Comment:
this is a breaking change we probably don't want to make

Reply:
Unfortunately, we may want to make a breaking change here. We want the user to specify the model when using the embeddings, as we want to encourage them to use the new models but we can't frequently update the default model here.
Do you have any suggestions?

Reply:
i would recommend at least giving folks a month or two heads up before doing this by adding a warning for now that the breaking change will happen by a certain date. here's an example: https://github.com/langchain-ai/langchain/pull/14614

Reply:
and for now update all of the documentation to show the latest models being passed in

Reply:
I agree that giving a warning could be a better idea.

Reply:
I moved the default value to `validate_environment`. Thus it should be backward compatible now.